airflow-core/src/airflow/utils/db.py

```diff
@@ -1212,6 +1212,22 @@ def downgrade(*, to_revision, from_revision=None, show_sql_only=False, session:
 
     log.info("Attempting downgrade to revision %s", to_revision)
     config = _get_alembic_config()
+    # Check if downgrade is less than 3.0.0 and requires that `ab_user` fab table is present
+    if to_revision < _REVISION_HEADS_MAP["3.0.0"]:
```

Comment:
Isn't this just comparing the migration ids as strings?

Reply:
Yeah, it's wrong. I will work it out